assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml

```diff
@@ -0,0 +1,130 @@
+    Properties: 
+      DBClusterParameterGroupName: 
+        Ref: RDSDBClusterParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
+      Engine: aurora
+      MasterUserPassword: 
+        Ref: password
+      MasterUsername: 
+        Ref: username
+      StorageEncrypted: false  
+    Type: "AWS::RDS::DBCluster"
+  RDSDBClusterParameterGroup: 
+    Properties: 
+      Description: "CloudFormation Sample Aurora Cluster Parameter Group"
+      Family: aurora5.6
+      Parameters: 
+        time_zone: US/Eastern
+    Type: "AWS::RDS::DBClusterParameterGroup"
+  RDSDBInstance1: 
+    Properties: 
+      AvailabilityZone: 
+        Fn::GetAtt:
+          - Subnet1
+          - AvailabilityZone
+      DBClusterIdentifier: 
+        Ref: RDSCluster
+      DBInstanceClass: db.r3.xlarge
+      DBParameterGroupName: 
+        Ref: RDSDBParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
+      Engine: aurora
+      PubliclyAccessible: "true"
+    Type: "AWS::RDS::DBInstance"
+  RDSDBInstance2: 
+    Properties: 
+      AvailabilityZone: 
+        Fn::GetAtt:
+          - Subnet2
+          - AvailabilityZone
+      DBClusterIdentifier: 
+        Ref: RDSCluster
+      DBInstanceClass: db.r3.xlarge
+      DBParameterGroupName: 
+        Ref: RDSDBParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
+      Engine: aurora
+      PubliclyAccessible: "true"
+    Type: "AWS::RDS::DBInstance"
+  RDSDBParameterGroup:
+    Type: 'AWS::RDS::DBParameterGroup'
+    Properties:
+      Description: CloudFormation Sample Aurora Parameter Group
+      Family: aurora5.6
+      Parameters:
+        sql_mode: IGNORE_SPACE
+        max_allowed_packet: 1024
+        innodb_buffer_pool_size: '{DBInstanceClassMemory*3/4}'                      
+
+---
+
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Creates RDS Cluster
+Resources:
+  RDSCluster_: 
+    Properties: 
+      DBClusterParameterGroupName: 
+        Ref: RDSDBClusterParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
+      Engine: aurora
+      MasterUserPassword: 
+        Ref: password
+      MasterUsername: 
+        Ref: username
+    Type: "AWS::RDS::DBCluster"
+  RDSDBClusterParameterGroup_: 
+    Properties: 
+      Description: "CloudFormation Sample Aurora Cluster Parameter Group"
+      Family: aurora5.6
+      Parameters: 
+        time_zone: US/Eastern
+    Type: "AWS::RDS::DBClusterParameterGroup"
+  RDSDBInstance1_: 
+    Properties: 
+      AvailabilityZone: 
+        Fn::GetAtt:
+          - Subnet1
+          - AvailabilityZone
+      DBClusterIdentifier: 
+        Ref: RDSCluster
+      DBInstanceClass: db.r3.xlarge
+      DBParameterGroupName: 
+        Ref: RDSDBParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
+      Engine: aurora
+      PubliclyAccessible: "true"
+    Type: "AWS::RDS::DBInstance"
+  RDSDBInstance2_: 
+    Properties: 
+      AvailabilityZone: 
+        Fn::GetAtt:
```

Comment:
Remove `GetAtt` function and leave it hardcoded:

`AvailabilityZone: eu-west-1b`